src/Middleware/ConcurrencyLimiter/src/ConcurrencyLimiterMiddleware.cs

```diff
@@ -48,19 +48,23 @@ public async Task Invoke(HttpContext context)
         {
             var waitInQueueTask = _queuePolicy.TryEnterAsync();
 
+            // Make sure we only ever call GetResult once on the TryEnterAsync ValueTask b/c it resets.
+            var result = false;
```

Comment:
nit: I think you don't need to set a default value of false here. Instead do: `bool result;`